shared/style/option_menu.css

```diff
@@ -13,3 +13,14 @@ form[role="dialog"][data-subtype="menu"].visible {
   transform: translateY(0);
 }
 
+/* ----------------------------------
+ * Dialogs trick: for further information
```

Comment:
proposition:

This CSS makes the application interaction-less during dialog animations, to prevent double clicking on elements.